### PR TITLE
CH-141: Prevent disabled filters from being toggled using keyboard

### PR DESCRIPTION
**Pull request by chrieinv**

### CODE REVIEW

File: src/scripts/Components/TabPanel/Reuse/FilterBar/FilterBar.js [hunk 1]
```diff
@@ -108,30 +108,50 @@ class FilterBar extends React.Component {
...
   }
 
+  /**
+   * Check if applying a filter results in content or not
+   * @param {string} filterGroup Key in filterCounts where the filter can be found, e.g. 'language'
+   * @param {string} filterItem The filter to check, e.g. 'nb'
+   * @return {boolean} true if applying filter option will result in content, false if it results in zero content
+   */
+  filterHasContent = (filterGroup, filterItem) => {
+    console.log(filterGroup, filterItem);
```
Comment (emmachughes) [inline src/scripts/Components/TabPanel/Reuse/FilterBar/FilterBar.js:118]: Leftover debugging statement

Comment (chrieinv) [reply src/scripts/Components/TabPanel/Reuse/FilterBar/FilterBar.js]: Fixed

File: src/scripts/Components/TabPanel/Reuse/FilterBar/FilterBar.js [hunk 2]
```diff
@@ -108,30 +108,49 @@ class FilterBar extends React.Component {
...
 
+  /**
+   * Check if applying a filter results in content or not
+   * @param {string} filterGroup Key in filterCounts where the filter can be found, e.g. 'language'
+   * @param {string} filterItem The filter to check, e.g. 'nb'
+   * @return {boolean} true if applying filter option will result in content, false if it results in zero content
+   */
+  filterHasContent = (filterGroup, filterItem) => {
+    const counts = this.state.filterCounts[filterGroup] ?? [];
+    return counts[filterItem] > 0 ?? false;
```
Comment (emmachughes) [inline src/scripts/Components/TabPanel/Reuse/FilterBar/FilterBar.js:119]: `??` doesn't do anything in this context. For a non-existent item, this is equivalent to `(undefined > 0) ?? false`, and comparing undefined against a number returns false, so the expression to the right of `??` is never evaluated.

**Review by emmachughes — APPROVED** [review]

File: src/scripts/Components/TabPanel/Reuse/FilterBar/FilterBar.js (hunk 2 above)
Comment (chrieinv) [reply src/scripts/Components/TabPanel/Reuse/FilterBar/FilterBar.js]: Fixed

**Pull request merged**